src/coreclr/src/vm/interoplibinterface.cpp

```diff
@@ -304,64 +307,62 @@ namespace
             }
             CONTRACT_END;
 
-            DWORD objCount;
-            DWORD objCountMax;
-
             struct
             {
                 PTRARRAYREF arrRef;
                 PTRARRAYREF arrRefTmp;
```

Comment:
Any reason we're not getting rid of `arrRefTmp`?

Reply:
Oops. Nope, just missed it.